models/namespace.go

```diff
@@ -126,6 +129,66 @@ func (p *Namespace) Verify() error {
 	return nil
 }
 
+// Decrypt decrypt user/password in namespace
+func (p *Namespace) Decrypt(key string) (err error) {
+	if !p.IsEncrypt {
+		return nil
+	}
+	// Users
+	for i := range p.Users {
+		p.Users[i].UserName, err = decrypt(key, p.Users[i].UserName)
+		if err != nil {
+			return
+		}
+		p.Users[i].Password, err = decrypt(key, p.Users[i].Password)
+		if err != nil {
+			return
+		}
+	}
+	// Slices
+	for i := range p.Slices {
+		p.Slices[i].UserName, err = decrypt(key, p.Slices[i].UserName)
+		if err != nil {
+			return
+		}
+		p.Slices[i].Password, err = decrypt(key, p.Slices[i].Password)
+		if err != nil {
+			return
+		}
+	}
+
+	return nil
+}
+
+// Encrypt encrypt user/password in namespace
+func (p *Namespace) Encrypt(key string) (err error) {
+	p.IsEncrypt = true
+	// Users
+	for i := range p.Users {
+		p.Users[i].UserName, err = encrypt(key, p.Users[i].UserName)
+		if err != nil {
+			return
+		}
+		p.Users[i].Password, err = encrypt(key, p.Users[i].Password)
+		if err != nil {
+			return
+		}
+	}
+	// Slices
+	for i := range p.Slices {
+		p.Slices[i].UserName, err = encrypt(key, p.Slices[i].UserName)
+		if err != nil {
+			return
+		}
+		p.Slices[i].Password, err = encrypt(key, p.Slices[i].Password)
+		if err != nil {
+			return
+		}
+	}
+
+	return nil
+}
+
```

Comment:
please add unit tests for Encrypt() and Decrypt()

Reply:
ok，i forget it

Reply:
done